### PR TITLE
Static file embedding for web

### PR DESCRIPTION
**Pull request by shellymathew98**
- Static files are embedded for web.
- Go embed is supported only by go versions 1.16 and above, thus edited the version in go.mod here.

### CODE REVIEW

File: web/routes.go [hunk 1]
```diff
@@ -15,14 +15,19 @@
 package web
```
Comment (manitgupta) [inline web/routes.go:15]: Can we also fix the `webv2` imports as part of this PR?

**Review by manitgupta — COMMENTED** [review]
Can you also add details around how you validated the change? 

An ideal test would be like so:
1. Build the binary
2. Copy it over to some unknown location
3. run the binary
4. the webUI should work as expected.

File: main.go [hunk 2]
```diff
@@ -39,6 +40,9 @@ import (
 	"github.com/google/subcommands"
 )
 
+//go:embed frontend/*
+var forntendDir embed.FS
```
Comment (manitgupta) [inline main.go:44]: nit: spelling of variable `frontendDir`

**Comment by shellymathew98** [comment]
> Can you also add details around how you validated the change?
> 
> An ideal test would be like so:
> 
> 1. Build the binary
> 2. Copy it over to some unknown location
> 3. run the binary
> 4. the webUI should work as expected.

- I tried placing the binary on a different location and webUI works fine from there.

File: web/routes.go (hunk 1 above)
Comment (manitgupta) [reply web/routes.go:15]: Sure.

**Review by manitgupta — APPROVED** [review]
LGTM

**Pull request merged**